Packs/Code42/Integrations/Code42/Code42.py

```diff
@@ -509,6 +513,27 @@ def alert_resolve_command(client, args):
         return_error(create_command_error_message(demisto.command(), e))
 
 
+def alert_search_command(client, args):
+    username = args["username"]
+    try:
+        alerts = client.search_alerts(username)
+        alert_context = []
+        for alert in alerts:
```

Comment:
room for list comprehension....

`alert_context = [map_to_code42_alert_context(alert) for alert in alerts]`